### PR TITLE
T018 clean footer

### PR DESCRIPTION
**Pull request by tiaruzica**

### CODE REVIEW

**Review by johnchen383 — APPROVED** [review]
LGTM. Feel free to merge to main :))

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -1,21 +1,46 @@
...
+import '../styles/footer.scss';
+import LOGO_BLACK from "../assets/CreateOpsYTech.png";
+import LINKEDIN from "../assets/linkedin.png";
+import FACEBOOK from "../assets/facebook.png";
+import INSTAGRAM from "../assets/instagram.png";
+import EMAIL from "../assets/email.png";
+import Properties from '../data/Properties';
+
+
+
```
Comment (KennethZhu904) [inline src/components/Footer.tsx:11]: Bit a too much space, pls remove them.

File: src/screens/Landing.tsx [hunk 2]
```diff
@@ -1,26 +1,45 @@
...
+                <h1>Image</h1>
+                <Image src="https://drive.google.com/file/d/1GD4DUn3ehKOqwz32R-tMjGnWSZw6wVlG/view" />
+            </Section>
+
+            {/*Video*/}
+            <Section>
+                <h1>YouTube Video</h1>
+                <YouTubeVideo src="https://www.youtube.com/watch?v=iy3VW4L0dAY" />
+            </Section>
+
```
Comment (KennethZhu904) [inline src/screens/Landing.tsx:38]: Just remove one space and that should be fine

File: src/widgets/Image.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+  //  Obtain google drive image ID. 
+  var id = parts[parts.length-2];
+
+  //  Construct the image URL for website display. 
+  var output = 'https://drive.google.com/uc?export=view&id=' + id;
+
+  return <img className="cust-image" alt="YTech NZ drive" src={output}/>
+}
+
```
Comment (KennethZhu904) [inline src/widgets/Image.tsx:18]: Remove one space and it should be fine.

File: src/widgets/YoutubeVideo.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  
+  var url = props.src;
+  
+  //  Split url to get video's embedded ID. 
+  var parts = url.split('=');
+  var vid_id = parts[parts.length-1];
+    
+  return <iframe
+    src = {`https://www.youtube.com/embed/${vid_id}`}
+    //className="cust_video"
```
Comment (KennethZhu904) [inline src/widgets/YoutubeVideo.tsx:13]: please get rid of that comment - if you don't need it anymore.

**Review by KennethZhu904 — APPROVED** [review]
LGTM - once you fix some of my changes, you can merge them :)

**Pull request merged**